include/vara/Feature/OrderedFeatureVector.h

```diff
@@ -26,6 +26,12 @@ class OrderedFeatureVector {
   /// Insert feature while preserving ordering.
   void insert(Feature *F);
 
+  void insert(const OrderedFeatureVector &OFV) {
+    for (const auto &F : OFV) {
+      insert(F);
+    }
+  }
+
```

Comment:
Together with the initializer list insert, we should create an insert that takes two iterators and just forward begin/end to it if we have a container insert like this or initializer list